drivers/SmartThings/jbl/src/fields.lua

```diff
@@ -0,0 +1,16 @@
+--- Table of constants used to index in to device store fields
+--- @module "fields"
+--- @class table
+--- @field IPV4 string the ipV4 address of the device
+
+local fields = {
```

Comment:
Style nit: `fields` should be capitalized. Won't hold up review for this, but would be nice to address if time permits.

Reply:
Done